sinks/src/main/scala/com.snowplowanalytics.snowplow.eventgen/Config.scala

```diff
@@ -59,16 +60,23 @@ object Config {
     case class Fixed(at: Instant) extends Timestamps
   }
 
-  case class Cli(config: Config, output: URI, region: Option[String])
+  case class Cli(config: Config)
 
-  /* Temporary class for raw parameters */
-  case class RawCli(config: Option[Path], output: URI, region: Option[String])
+  case class Output(
```

Comment:
If we were to be consistent with how we do things in every other Snowplow application.....

Then we would write make `Output` the sealed trait.  And `File`, `Kinesis`, `Kafka` and `Pusbub` would be classes of trait `Output`.

Then, the configuration file would look like this:

```
output: {
  type: "File"
  path: "file:/path/to/my-events"
}
```

..or...

```
output: {
  type: "Kafka"
  topic: "my-topic"
  // etc
}
```

etc.

And it would mean we don't need the `throw new RuntimeException` if the `output` block contains more than sink.

I don't so much mind how it's done here, but it does stand out as being inconsistent.  And I think it's a fairly easy code change to make the code compatible with our other apps.

Reply:
OK as you'll see by the two commits I added since commenting on the main thread of the PR, I'm now confused about what I did and how it realates to what the config looks like. 

I'm going to stop tinkering and just wait till I get some feedback - fairly likely I didn't get it right but think I could do with a steer on where I'm off the mark. :)